actionpack/lib/abstract_controller/base.rb

```diff
@@ -96,7 +97,6 @@ def action_methods
             public_instance_methods(false))
 
           methods.map!(&:to_s)
-
```

Comment:
There are still unnecessary changes in this file - we try to avoid those to keep the git history clean.

Can you also clean up the commit message, it contains all of the squashed commits' messages (e.g. "Add suggested changes from feedback").